modules/web/src/app/dynamic/enterprise/cluster-backups/list/restore/delete-dialog/template.html

```diff
@@ -0,0 +1,62 @@
+<!--
+               Kubermatic Enterprise Read-Only License
+                      Version 1.0 ("KERO-1.0”)
+                  Copyright © 2023 Kubermatic GmbH
+
+1. You may only view, read and display for studying purposes the source
+   code of the software licensed under this license, and, to the extent
+   explicitly provided under this license, the binary code.
+2. Any use of the software which exceeds the foregoing right, including,
+   without limitation, its execution, compilation, copying, modification
+   and distribution, is expressly prohibited.
+3. THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND,
+   EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+   MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+   IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+   CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+   TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+   SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+END OF TERMS AND CONDITIONS
+-->
+
+<div id="km-delete-project-dialog"
+     (keydown.enter)="onEnterKeyDown()">
+  <km-dialog-title>Delete Restore</km-dialog-title>
+  <mat-dialog-content>
+    <p *ngIf="restores.length === 1">
+      Delete
+      <b ngxClipboard
+         [cbContent]="restores[0].name"
+         matTooltip="Click to copy"
+         class="km-copy">{{restores[0].name}}</b>
+      restore permanently?
+    </p>
+    <p *ngIf="restores.length > 1">
+      Type
+      <b ngxClipboard
+         [cbContent]="'yes'"
+         matTooltip="Click to copy"
+         class="km-copy">yes</b>
+      to delete all selected restores permanently?
+    </p>
+    <mat-form-field>
+      <mat-label>{{restores.length > 1 ? "Submit restores deletion" : "restore Name" }}</mat-label>
+      <input id="km-delete-project-dialog-input"
+             matInput
+             type="text"
+             title="Project Name"
```

Comment:
Please also replace other usages of `project` with `restore` in this file.
```suggestion
             title="Restore Name"
```